src/modules/fundingManager/bondingCurve/FM_BC_Bancor_Redeeming_VirtualSupply_v1.sol

```diff
@@ -598,4 +598,27 @@ contract FM_BC_Bancor_Redeeming_VirtualSupply_v1 is
             );
         }
     }
+
+    //--------------------------------------------------------------------------
+    // Overridden Internal Functions
+
+    /// @notice    Distributes issuance tokens by minting them to the receiver.
+    /// @param  _receiver The address that will receive the bought tokens.
+    /// @param  _issuanceMintAmount The amount of issuance tokens to distribute.
+    function _handleIssuanceTokenAfterBuy(
```

Comment:
In the events etc we treat this as plural (i.e. `TokensBought` and not `TokenBought`), maybe we should also name these `_handleIssuanceTokensAfterBuy` instead of the singular version? (also the collateral version)